packages/compass/src/main/window-manager.ts

```diff
@@ -109,9 +158,10 @@ function showConnectWindow(
     }
   > = {}
 ): BrowserWindow {
+  // Get saved window bounds
+  const { isMaximized, ...bounds } = getSavedWindowBounds(compassApp);
   const windowOpts = {
-    width: Number(DEFAULT_WIDTH),
-    height: Number(DEFAULT_HEIGHT),
+    ...bounds,
```

Comment:
Does electron gracefully handle tries to open a window that's larger than their screen size or out of bounds of their current screen? I'm wondering if we need to account for situations where a user goes from a larger monitor to a smaller one.